Basic-Car-Maintenance/Shared/Localizable.xcstrings

```diff
@@ -43,29 +67,48 @@
     },
     "Add Vehicle" : {
       "localizations" : {
-        "de" : {
+        "cs" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Fahrzeug hinzufügen"
+            "value" : "Přidat vozidlo"
```

Comment:
Is this overwriting the German translation?

Reply:
Oh, it seems like I must’ve missed a line! Will fix that tomorrow. Sorry!

Reply:
Fixed.